.github/workflows/test.yml

```
@@ -110,48 +110,8 @@ jobs:
       - name: Unit Tests
         run: sbt ++${{ matrix.scala }} standardLibrary/test
 
-  website-api-docs:
-    name: API Docs
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        project: [chisel3]
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Setup Scala
-        uses: olafurpg/setup-scala@v10
-        with:
-          java-version: adopt@1.8
-      - name: Cache Scala
-        uses: coursier/cache-action@v5
-      - name: Calculate API Docs Cache Key
-        id: calc-key
-        run: |
-          KEY=$(make -C website --always-make --dry-run apis-${{ matrix.project }} | sha256sum | awk '{print $1}')
-          echo "::set-output name=cache_key::${KEY}"
-      - name: Cache API Docs
-        uses: actions/cache@v2
-        with:
-          path: website/build/api/${{ matrix.project }}
-          key: ${{ runner.os }}-${{ matrix.project }}-${{ steps.calc-key.outputs.cache_key }}
-          # On miss, still start from previous value for incremental rebuild
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.project }}
-      - name: ${{ matrix.project }} APIs
-        run: make -C website apis-${{ matrix.project }}
-      - name: Tar build artifacts
-        run: tar zcf website/build/api/${{ matrix.project }}.tar.gz website/build/api/${{ matrix.project }}
-      - name: Share API Docs
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.project }}-api-docs
-          path: website/build/api/${{ matrix.project }}.tar.gz
-
   website:
-    name: Build & Deploy Website
-    needs: [website-api-docs]
+    name: Build Website
```

```suggestion
    name: Build Website (and API Docs)
```

Please call out that building the website also builds the API docs--I will forget and then add a new test in the future if this isn't mentioned 🙂

---

its not the API docs, it's the Mdocs, do you agree?